rules/fon/fon-alt.js

```diff
@@ -0,0 +1,34 @@
+
+( function ( $ ) {
+	'use strict';
+
+	var fonAlt = {
+		id: 'fon-alt',
+		name: 'fon-alt',
+		description: 'Fon input keyboard',
+		date: '2018-05-18',
+		URL: 'http://github.com/wikimedia/jquery.ime',
+		author: 'Mahuton POSSOUPE',
+		license: 'GPLv3',
+		version: '1.0',
+		patterns: [
+<<<<<<< HEAD
```

Comment:
please fix these merge conflict markers

Reply:
I already fixed this with @Mahuton on his laptop. He just needs to upload the new version :)